CHANGELOG.md

```diff
@@ -6,7 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 ### Changed
-- Upgraded `jackson-databind` to 2.9.7 (was 2.9.0)
+- Upgraded `jackson-databind` to 2.9.7 (was 2.9.0).
+- Upgraded `hive-jdbc` to 2.3.4 (was 2.0.0).
```

Comment:
Hopefully this change is backwards compatible with how it's currently being used in production, will need to keep an eye on this when a new release is done and used for the first time. Ideally we'd test a SNAPSHOT build first.